src/dotnet-svcutil/lib/tests/src/GlobalToolTests.cs

```diff
@@ -111,7 +111,7 @@ public void FullFramework(string testCaseName)
         [InlineData("netstd20")] //WCF package older than V4.10 get referened and CloseAsync() is generated
         [InlineData("net60")] //WCF package newer than V4.10 get referened and CloseAsync() is not generated
         [InlineData("net60net48")] //WCF package newer than V4.10 and System.ServiceModel.dll are referenced conditionally by target and CloseAsync() be generarted with conditional compilation mark
-        public void MultiTargetCloseAsyncGeneration(string testCaseName)
+        public async void MultiTargetCloseAsyncGeneration(string testCaseName)
         {
```

Comment:
Switch to `async Task` and not `async void` otherwise xunit doesn't see failures after the first async await